test/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java

```diff
@@ -120,7 +129,35 @@ public void successfulDeletePermanently() {
         verify(mockSurveyDao).deleteSurveyPermanently(keysCaptor.capture());
         assertEquals(survey, keysCaptor.getValue());
     }
-    
+
+    @Test(expected = BadRequestException.class)
+    public void logicallyDeleteSurveyNotEmptySharedModules() {
+        when(mockSharedModuleMetadataService.queryAllMetadata(anyBoolean(), anyBoolean(), anyString(), anySetOf(String.class)))
+                .thenReturn(ImmutableList.of(makeValidMetadata()));
```

Comment:
I can't verify anything right after a method throws an exception. Move those verifications to success cases.